src/base_multipart.rs

```diff
@@ -88,4 +103,28 @@ mod tests {
             .send()
             .await;
     }
+
+    #[test]
+    fn deref() {
```

Comment:
I would call this one `test_deref` just for the sake of consistency with the other tests.